### ISSUE TITLE
Link name does not have discernible text on record show page

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/pulmap#992]
### Error in spec/features/accessibility_spec.rb
accessibility when visiting record show page complies with WCAG

       link-name: Links must have discernible text (serious)
       https://dequeuniversity.com/rules/axe/4.3/link-name?application=axeAPI
       The following 1 node violate this rule:
           
               Selector: .pu-logo
               HTML: <a class="pu-logo" href="http://www.princeton.edu"><img width="200" src="/assets/pu-51a408b14408e04caba9f75a3d4b0e92f7577e4b7aa00efd0fa1b7efd52b629a.png"></a>
               Fix all of the following:
               - Element is in tab order and does not have accessible text
               Fix any of the following:
               - Element does not have text that is visible to screen readers
               - aria-label attribute does not exist or is empty
               - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
               - Element has no title attribute
